cfgov/unprocessed/js/modules/transition/BaseTransition.js

```diff
@@ -97,7 +96,7 @@ function BaseTransition( element, classes ) { // eslint-disable-line max-stateme
     _dom.style.oTransitionDuration = '0';
     _dom.style.transitionDuration = '0';
     _dom.removeEventListener( _transitionEndEvent,
```

Comment:
L:98 and 115 all have their first param on the same line as the function and should break to a new line like the one farther above to keep consistent.